taskiq/schedule_sources/label_based.py

```diff
@@ -1,9 +1,12 @@
+from logging import getLogger
 from typing import List
 
 from taskiq.abc.broker import AsyncBroker
 from taskiq.abc.schedule_source import ScheduleSource
 from taskiq.scheduler.scheduler import ScheduledTask
 
+logger = getLogger(__name__)
```

Comment:
What name is it going to be? If it starts with taskiq (it probably should), then it's ok.

Reply:
I checked it. The name is ok.